### PR TITLE
[memtracker](accuracy) should not account resuable buffer to query memtracker

### PR DESCRIPTION
**Pull request by yiguolei**
## Proposed changes
1. should only account resuable buffer to global compression tracker.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yiguolei** [comment]
run buildall

File: be/src/util/block_compression.cpp [hunk 1]
```diff
@@ -845,9 +865,13 @@ class ZstdBlockCompression : public BlockCompressionCodec {
             compressed_buf.data = reinterpret_cast<char*>(output->data());
             compressed_buf.size = max_len;
         } else {
-            // reuse context buffer if max_len <= MAX_COMPRESSION_BUFFER_FOR_REUSE
-            context->buffer.resize(max_len);
-            compressed_buf.data = reinterpret_cast<char*>(context->buffer.data());
+            {
+                SCOPED_SWITCH_THREAD_MEM_TRACKER_LIMITER(
```
Comment (xinyiZzz) [inline be/src/util/block_compression.cpp:869]: delete line:849

**Comment by yiguolei** [comment]
run buildall

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Review by mrhhsg — APPROVED** [review]
LGTM

**Pull request merged**